test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -91,6 +91,20 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'unifiedSearch.autocomplete.valueSuggestions.tiers (array)',
         'unifiedSearch.autocomplete.valueSuggestions.timeout (duration)',
         'data.search.aggs.shardDelay.enabled (boolean)',
+        'data.search.sessions.cleanupInterval (duration)',
+        'data.search.sessions.defaultExpiration (duration)',
+        'data.search.sessions.enabled (boolean)',
+        'data.search.sessions.expireInterval (duration)',
+        'data.search.sessions.management.expiresSoonWarning (duration)',
+        'data.search.sessions.management.maxSessions (number)',
+        'data.search.sessions.management.refreshInterval (duration)',
+        'data.search.sessions.management.refreshTimeout (duration)',
+        'data.search.sessions.maxUpdateRetries (number)',
+        'data.search.sessions.monitoringTaskTimeout (duration)',
+        'data.search.sessions.notTouchedInProgressTimeout (duration)',
+        'data.search.sessions.notTouchedTimeout (duration)',
+        'data.search.sessions.pageSize (number)',
+        'data.search.sessions.trackingInterval (duration)',
```

Comment:
I see you already added the right deprecations for this, and you already have a TODO to update the allow-list in Cloud afterwards :+1:

Don't forget to update `src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker` in this PR, though!